docs/book/reference/zenml.md

```diff
@@ -0,0 +1,25 @@
+# 🙏 Association with ZenML
+
+[![maintained-by-zenml](https://user-images.githubusercontent.com/3348134/173032050-ad923313-f2ce-4583-b27a-afcaa8b355e2.png)](https://github.com/zenml-io/zenml)
+
+It is not necessary to use the MLOps stacks recipes presented here alongside the
+[ZenML](https://github.com/zenml-io/zenml) framework, but it is highly
+recommended to do so. The ZenML framework is designed to be used with these
+recipes, and the recipes are designed to be used with ZenML.
+
+However, ZenML works seamlessly with the infrastructure provisioned through
```

Comment:
```suggestion

```
I think this sentiment is covered well in the previous paragraph already and we can remove this line.